modules/travel_pay/app/controllers/travel_pay/application_controller.rb

```diff
@@ -8,8 +8,33 @@ class ApplicationController < ::ApplicationController
 
     protect_from_forgery with: :exception
 
+    before_action :authenticate
+
+    ##
+    # This before_action is feature flag driven and should be retired
+    #
+    # Flag name: travel_pay_power_switch
+    # Intent:    allow env-specific request blocking
+    #
+    # Retirement conditions:
+    # * Feature is in production
+    # * Finer-grained feature flags exist as needed
+    #
+    # Upon retirement:
+    # * Remove feature flag from config/features.yml
+    # * Remove feature flag from database (requires platform support for prod)
+    # * Remove this before_action
+    # * Remove block_if_flag_disabled definition
+    # * Remove flag_conditions_met? definition
```

Comment:
Should remove this line. I shifted to a simpler way, and removed the code this corresponded to.